crates/bevy_input/src/input.rs

```diff
@@ -20,6 +20,14 @@ use bevy_ecs::schedule::State;
 /// * [`Input::just_pressed`] will return `true` for one frame after a press event.
 /// * [`Input::just_released`] will return `true` for one frame after a release event.
 ///
+/// ## Input types
+///
+/// This crate provides support for the following parameters of `Input`:
+///
+/// * [`KeyCode`](crate::KeyCode) and [`ScanCode`](crate::ScanCode) for keyboard input
```

Comment:
```suggestion
/// ## ButtonInput types
///
/// This crate provides support for the following parameters of `ButtonInput`:
///
/// * [`KeyCode`](crate::KeyCode)  for keyboard input
```
`Input` has since been renamed `ButtonInput` and `ScanCode` removed with `KeyCode` taking it's place as the physical key. (I do not remember if `Key`, the present logical key, is relevant to `ButtonInput` at all).